src/Microsoft.AspNet.Http/Features/HttpRequestFeature.cs

```diff
@@ -12,7 +12,7 @@ public class HttpRequestFeature : IHttpRequestFeature
     {
         public HttpRequestFeature()
```

Comment:
FYI: This implementation is used exclusively in unit tests, no optimizations required. We should add a doc comment to that effect.